tests/test_cli.py

```diff
@@ -65,30 +75,28 @@ def runner():
 def test_success_download(runner, mock_get_datasets, mock_requests_file_get):
     print(f"Try the get_datasets() mock: {machine_data_hub.cli.get_datasets('hello')}")
     print(f"Try the request.get mock: {requests.get('hello').json()}")
-    result = runner.invoke(app, ["download", "Combined Cycle Power Plant Data Set"])
+    result = runner.invoke(app, ["download", 1, 1])
     assert result.exit_code == 0
 
 
-def test_fail_download(runner, mock_requests_get):
+def test_fail_download(runner, mock_requests_get, mock_requests_file_get):
     # passing in incorrect name
-    result = runner.invoke(app, ["download", "Combined Cycle Power Plant Data"])
+    result = runner.invoke(app, ["download", 9999])
     assert result.exit_code == 0
 
 
-def test_metadata(runner, mock_requests_get):
-    result = runner.invoke(app, ["metadata", "Combined Cycle Power Plant Data Set"])
+def test_metadata(runner, mock_requests_get, mock_requests_file_get):
+    result = runner.invoke(app, ["metadata", 1])
     assert result.exit_code == 0
     # assert f"Downloading {name} right now!" in result.stdout
 
 
 def test_suggest(runner):
-    result = runner.invoke(app, ["suggest",
-                                 "--link", "www.google.com",
-                                 "--name", "Test",
-                                 "--summary", "Testing summary"])
-    assert result.exit_code == 0
+    print("don't want to test every time")
```

Comment:
We can make a mock for the `POST`. understand not wanting to cause a side effect of posting a suggested dataset everytime you run. Please add an issue to mock suggest post and leave this as is for this PR.